### DISCUSSION TITLE
Git author must have access to the project Vercel

### DISCUSSION THREAD

**Discussion by hembo** [vercel/community#2199] [Help] [answered]
Hello, I just discovered that in June 22 was the last successful deployment to my Vercel account. All commits to Bitbuckets  triggers the deployment but i see error message "Git author must have access to the project Vercel" (which i have) and when i click the link i am taken to team creation page. But why do i need to create the team?

**Comment by mcsdevv** [comment, marked as answer]
Hi @hembo, this error occurs when the Git authors' email is not the same as the email address associated with the Vercel Personal Account.

I recommend checking the two to see which does not match, you can find information on doing so in [this support article](https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata).

Edit: you are only seeing this now as a minor security update was made for this.

**Reply by hembo** [reply]
Hello,

Sadly this is not the issue. Then the error is "Commit author needs to be Bitbucket account"

Hembo

**Reply by hembo** [reply]
Just to close the topic ... i joined the payed service and the all worked ...
So it would be good to have more clear error messages.

**Comment by aceaspades-worldspark** [comment]
Cheers. Question to piggyback on this one. I have two Github accounts, one personal and one for work. I've added both to the Github organization, and both are admins of my repo-in-question (but only my work account is a member of the Vercel Team, it's a paid Pro account managed by my company). 

I'm getting this error when _either_ contributing accounts push a commit to the master branch. Neither can initiate a build to production by pushing to main. I also attempted to push to a staging branch (which luckily built a Preview Deployment), then I merged a PR into main using my work account, but the error persists.

**Reply by amyegan** [reply]
Hi @aceaspades-worldspark. Have you verified that your email addresses match? https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel#correctly-configured-metadata

**Reply by aceaspades-worldspark** [reply]
I have definitely. I've recorded a Loom to show the problem in greater detail. https://www.loom.com/share/2db3818184234384b5b4d3650aef0cda

- I am the owner of the repo
- I am the owner of the Vercel team
- The project was originally published by my personal account, but I've removed it from everywhere, except commit history. 
- Although I'm supposedly properly registered and connected between Vercel and the repo, I still cannot get the commit to properly trigger a build.

This is a production website, so I cannot take it offline.

**Reply by amyegan** [reply]
It looks like you already went through the documentation on [deploying private git repositories](https://vercel.com/docs/concepts/git#deploying-private-git-repositories) and set up your account correctly from what I can see in your video. Something strange seems to be happening here. 

Please open a new case at https://vercel.com/support/request with these details. That will help our team work with you more closely and keep your private info out of this public discussion.

**Comment by flashjames** [comment]
I have the same error, and it happened after I moved project to pro plan

**Reply by amyegan** [reply]
Hi @flashjames! Since you just moved the project to a pro plan, it could be that the permissions didn't make it through the transfer. If you disconnect and reconnect the repo on Git tab in Project Settings, that may get everything connected again.

If you tried that and it's still not working for you, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. They'll be able to discuss account details with you privately to help solve the problem.
